package.json

```diff
@@ -14,7 +14,7 @@
         "start": "npx tsdx watch --transpileOnly --noClean",
         "build": "npx tsdx build",
         "test": "npx tsdx test test/",
-        "test:integration": "jest --config ./test-integration/jest.config.js",
+        "test:integration": "jest --detectOpenHandles --config ./test-integration/jest.config.js",
```

Comment:
- Attempt to collect and print open handles preventing Jest from exiting cleanly 
- in cases where you need to use --forceExit in order for Jest to exit to potentially track down the reason